plugins/woocommerce/patterns/page-coming-soon-default.php

```diff
@@ -15,8 +15,8 @@
 
 ?>
 
-<!-- wp:woocommerce/coming-soon {"comingSoonPatternId":"page-coming-soon-default","className":"woocommerce-coming-soon-entire-site","style":{"color":{"background":"#bea0f2","text":"#000000"},"elements":{"link":{"color":{"text":"#000000"}}}}} -->
-<div class="wp-block-woocommerce-coming-soon woocommerce-coming-soon-entire-site has-text-color has-background has-link-color" style="color:#000000;background-color:#bea0f2"><!-- wp:cover {"customOverlayColor":"transparent","isUserOverlayColor":true,"minHeight":100,"minHeightUnit":"vh","className":"coming-soon-is-vertically-aligned-center coming-soon-cover","style":{"spacing":{"padding":{"top":"0px","bottom":"0px","left":"24px","right":"24px"}},"color":{"text":"inherit"},"elements":{"link":{"color":{"text":"inherit"}}}},"layout":{"type":"constrained","wideSize":"1280px"}} -->
+<!-- wp:woocommerce/coming-soon {"comingSoonPatternId":"page-coming-soon-default","className":"woocommerce-coming-soon-default","style":{"color":{"background":"#bea0f2","text":"#000000"},"elements":{"link":{"color":{"text":"#000000"}}}}} -->
```

Comment:
Updated classname `woocommerce-coming-soon-default ` to prevent conflicting styles with older version